content/content.md

```diff
@@ -1,68 +1,68 @@
 
-The Fourier transform is a way for us to take the combined wave, and get each of the sine waves back out. In this example, you can almost do it in your head, just by looking at the original wave.
+傅立葉轉換是讓我們從組合的波形中，取出每個正弦波的一種方式。在範例中，只需看著原始波形，您幾乎就可以想像得出來。
 
-Why? Turns out a lot of things in the real world interact based on these sine waves. We usually call them the wave's frequencies.
+為什麼？事實證明，現實世界中的許多事物都是基於這些正弦波進行交互的。 我們通常稱它們為波的頻率。
 
-The most obvious example is sound – when we hear a sound, we don’t hear that squiggly line, but we hear the different frequencies of the sine waves that make up the sound.
+最明顯的例子是聲音。當我們聽到聲音時，沒有聽到那條彎彎曲曲的線條，但是我們聽到了構成聲音中不同頻率的正弦波。
 
 <button id="together-button" class="button">Play Full Wave</button>
 
 <button id="split-button-1" class="button">Play High Frequency</button>
 
 <button id="split-button-2" class="button">Play Low Frequency</button>
 
-Being able to split them up on a computer can give us an understanding of what a person actually hears. We can understand how high or low a sound is, or figure out what note it is.
+在電腦上將音頻分開後，能夠使我們聽到個別實際的聲音。如此可以了解聲音的高低，也可以知道有哪些音符。
 
-We can also use this process on waves that don't look like they're made of sine waves.
+相同的處理方式，可以應用在正弦波以外的波形上。
 
-Let's take a look at this guy. It’s called a square wave.
+底下這種波形，叫做方波：
 
 <canvas id="square-wave" class="sketch" width=500 height=300></canvas>
 
-It might not look like it, but it also can be split up into sine waves.
+雖然它看起來是方方正正的波形，但實際上確實能夠分解成許多的正弦波。換句話說，方波是能夠由多個正弦波組合而成的！
 
 <canvas id="square-wave-split" class="sketch" width=500 height=500></canvas>
 
-We need a lot of them this time – technically an infinite amount to perfectly represent it. As we add up more and more sine waves the pattern gets closer and closer to the square wave we started with.
+這次我們分層解說。從技術上來說，波形可以是無限多層疊加出來的，然而隨著我們有越來越多的正弦波相加，波形會逐漸地和一開始的方波越來越接近。
 
 <canvas id="square-wave-build-up" class="sketch" width=500 height=500></canvas>
 <input id="square-wave-build-up-slider" type="range" min="0" max="1" value="0" step="any" >
 
 <button id="square-wave-button" class="button">Play Wave</button>
 
-*Drag the slider above to play with how many sine waves there are.*
+*試著滑動看看，觀察每層正弦波相加後得到的波形會是如何。*
 
-Visually, you'll notice that actually the first few sine waves are the ones that make the biggest difference. With the slider halfway, we have the general shape of the wave, but it's all wiggly. We just need the rest of the small ones to make the wigglyness flatten out.
+從視覺上來看，您會注意到實際上前幾個正弦波是最大的正弦波。將滑塊移到一半時，我們具有方波的大致形狀，但波形上方有許多看似小幅擺動的鋸齒狀圖案。我們只需要將其餘的小正弦波相加後就可以使方波上的擺動變得平坦。
 
-When you listen to the wave, you'll hear the sound get lower, because we're removing the higher frequencies.
+當您波放這個波形時，您會聽到很低的聲音，因為我們去除了聲音中較高的頻率。
 
-This process works like that for any repeating line. Give it a go, try drawing your own!
+這樣的方法，能夠處理任何具有週期性的波形。試試吧，嘗試自己繪製一個波形！
 
 <div class="multi-container">
 <div class="sketch" >
```

Comment:
GitHub doesn't let me comment on this line because it's not near a line you've edited, but just below, in the following line:

`<p id="wave-draw-instruction" class="instruction wave-instruction">Draw here!</p>`

Draw here! needs to be translated